NedyUdombat/Role-Assigner
# #164804962 Setup entry point for app

#### What does this PR do?
Setup entry point for app

#### Description of Task to be completed?
* setup app to use cors, body-parser
* install nodemon for development enviroment
* create basic MVC structure
* write test for endpoints

#### How should this be manually tested?
* clone `https://github.com/NedyUdombat/Role-Assigner.git`
* checkout to  a new branch = `ch-setup-express-mvc-env-164804962`
* pull from the remote branch to update the local branch
* run `npm install`
* run `cp .env.example .env`
* create a db
* fill in the details in your env
* run `sequelize db:migrate` (make sure you have sequelize installed globally)
* run `npm test`
or 
* run `npm run watch`
* visit
  * `/` & `/api/v1` on you REST API client (e.g POSTMAN)


#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#164804962

#### Screenshots (if appropriate)
none

#### Questions:
None.
